jsonschema/scripts/codegen.rb

```diff
@@ -214,6 +214,33 @@ def format_description(raw_description, indent_string: "")
   end
 end
 
+class Dotnet < Codegen
+  def initialize(paths)
+    language_type_by_schema_type = {
+      'integer' => 'long',
```

Comment:
Are we sure that the 'integer' has to be a long in our case? Wouldn't `int` be enough?

@mpkorstanje Is there an explicit reason for requiring a 8 byte number for the messages? As far as I understand, we use numbers for line can column positions, where the maximum of the 4-byte int 2,147,483,647, should be more than sufficient. Even Visual Studio [treats line numbers as 4 byte `int`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.text.itextsnapshotline.linenumber?view=visualstudiosdk-2022#microsoft-visualstudio-text-itextsnapshotline-linenumber). Working with `long` is cumbersome in .NET because all other API uses `int` and converting them back and force is painful.

So my suggestion would be to use `int` here.

Reply:
The same script generates the TimeStamp class which uses two longs to represent the number of seconds that have passed since 1 Jan 1970 and the number of partial seconds expressed as nanoseconds.
Using a signed int would work, but only for the next 13.5 years before the seconds field would overflow.
Would you all prefer that the expressive range of implementations be consistent? 
Would you prefer that we use Int32 for Location fields but Int64 for timestamp? (would require some fiddling with the script but I think do-able).

Reply:
I reckon the correct way of solving this would be to add the [`minimum` and maximum` keywords](https://json-schema.org/understanding-json-schema/reference/numeric#range) to the line and column fields. Then the code generation can determine the most appropriate type based on that.

Reply:
@gasparnagy Is it worth waiting for this change to the schema to move through the ecosystem? I'll do what ever you two think best.

Reply:
No. Let's just keep them as `long`. It does not worth the effort.